src/protocols/LoRaWAN/LoRaWAN.cpp

```diff
@@ -55,12 +55,97 @@ int16_t LoRaWANNode::begin() {
     return(RADIOLIB_ERR_NETWORK_NOT_JOINED);
   }
 
+  uint16_t nvm_table_version = mod->hal->getPersistentParameter<uint32_t>(RADIOLIB_PERSISTENT_PARAM_LORAWAN_VERSION_ID);
```

Comment:
This seems unused in this PR, is the intention to use it later?

Reply:
Yes, if future versions decide that the layout or size of some parts need to be shuffled, that parameter can be used to 'transition' from one version to another while keeping the session alive which in some cases could be crucial.

Reply:
OK, in that case could you please add a comment stating this, as well as to explicitly cast the variable to void so that it is not throwing an unused variable warning?